lib_xua/src/core/endpoint0/xua_endpoint0.c

```diff
@@ -490,6 +508,7 @@ void XUA_Endpoint0_init(chanend c_ep0_out, chanend c_ep0_in, NULLABLE_RESOURCE(c
         outuint(c_aud_ctl, AUDIO_STOP_FOR_DFU);
         /* No Handshake */
         DFU_mode_active = 1;
+        SetDFUFlag(0); // Clear the DFU mode flag so that a device reboot without setting it back again transitions back to runtime mode
```

Comment:
Can we just make this comment a bit more clear?

Reply:
I've removed the setting of DFUFlag to 0 here. I should just power cycle the device..

Reply:
okay! I think the change was good, just didn't understand the comment :)